whatsapp_sdk.gemspec

```diff
@@ -44,7 +44,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency("minitest", "~> 5.0")
   spec.add_development_dependency("rake", "~> 12.3")
 
-  spec.add_dependency("faraday", "~> 2")
+  spec.add_dependency("faraday", "~> 2.0", "> 2.0.1")
```

Comment:
I forced the version to be higher than 2.0.1 since 2.0.0 has a bug https://github.com/lostisland/faraday/issues/1362 ([see error failing](https://app.circleci.com/pipelines/github/ignacio-chiazzo/ruby_whatsapp_sdk/478/workflows/2aec4c47-65e2-4ea6-9a42-5a48dee65374/jobs/463))